backend/amd64/emit.mlp

```diff
@@ -757,7 +771,10 @@ let emit_call_probe_handler_wrapper i ~probe_label =
        encoding which produces an incorrect relocation and changes the meaning
        of the program. *)
     (* Emit the required encoding of "cmp $0, %eax" directly, using .byte *)
-    D.byte (Const 0x3dL);
+    if enabled_at_init then
+      D.byte (Const 0xe8L)
```

Comment:
Is the comment still correct? `e8` does not
look like the start of a `cmp` instruction.

Reply:
fixed